src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/DefaultProjectChangePublisher.cs

```diff
@@ -8,16 +8,17 @@
 using System.Composition;
 using System.IO;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed.Serialization;
+using Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project;
+using Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Serialization;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 
 namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin;
 
 [Shared]
-[Export(typeof(ProjectChangePublisher))]
-[Export(typeof(IOmniSharpProjectSnapshotManagerChangeTrigger))]
-internal class DefaultProjectChangePublisher : ProjectChangePublisher, IOmniSharpProjectSnapshotManagerChangeTrigger
+[Export(typeof(IProjectChangePublisher))]
```

Comment:
`IProjectChangePublisher` had to become an interface to allow `IOmniSharpProjectSnapshotManagerChangeTrigger` to become abstract. 😮‍💨